book/getting-started.rst

```diff
@@ -54,6 +44,17 @@ This command will bootstrap a new project in the directory ``my-project``.
                 - en
                 - de
 
+    Available languages are shown on `Crowdin`_.
+    Afterwards the languages had been activated run the following command:
+
+    .. code-block:: bash
+
+        # download all configured languages
+        bin/console sulu:admin:download-language
+
+        # download specific language
+        bin/console sulu:admin:download-language <language>
```

Comment:
Do we still really need that? :thinking: I think at least in the documentation the first command is more than enough. Don't want to bloat the getting started guide too much.

Reply:
sounds good for me